crates/pallet-evm-accounts-mapping/src/lib.rs

```diff
@@ -2,7 +2,7 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
-use frame_support::pallet_prelude::*;
+use frame_support::{inherent::Vec, pallet_prelude::*};
```

Comment:
Why `inherent`?

Reply:
Thanks, https://github.com/humanode-network/humanode/pull/940/commits/6a55d0f93837de61078a63e01572b502a503336b

Reply:
We need to import Vec explicitly as `GenesisConfig` is enabled in `no_std` now due to https://github.com/paritytech/substrate/pull/14108.

Thanks for mentioning, imported from `sp_std::prelude`.